slack_bolt/app/app.py

```diff
@@ -735,7 +735,7 @@ def step(
         elif not isinstance(step, WorkflowStep):
             raise BoltError(f"Invalid step object ({type(step)})")
 
-        self.use(WorkflowStepMiddleware(step, self.listener_runner))
+        self.use(WorkflowStepMiddleware(step))
```

Comment:
While steps from apps is no longer supported, this change shows the benefit well